monitoring/README.md

```diff
@@ -1,23 +1,57 @@
 ### Launch Prometheus & Grafana Locally
 
 ```bash
-docker-compose build
-docker-compose up -d prometheus grafana
-./grafana/bin/create-data-sources.sh
-./grafana/bin/upload-dashboards.sh
+A run monitoring up
+
+# A run monitoring down
+```
+
+Access Grafana by visiting:
+
+* http://${IP}:80
+
+Access Prometheus by visiting:
+
+* http://${IP}:9090/targets
+
+#### Notes:
+
+* `https` is not supported.
+* The default credentials are `admin`/`admin`.
+* The password must be changed on first login to something other than `admin`
+  or click on the `Skip` link when prompted to change your password.
+
+If the password is changed, for ease of future development, add the password to your
+`~/.profile`:
+
+```bash
+GRAFANA_PASS=xxxxx
+
+echo "export GRAFANA_PASS=${GRAFANA_PASS}" >> ~/.profile
 ```
 
-Ports:
+### Adding New Targets
 
-* Prometheus binds to :9090
-* Grafana binds to :3000
+`./prometheus/generate-prom.js` generates `prometheus.yml` within the
+container at build time.
+
+For local development, start by modifying
+`./prometheus/ymls/local.yml`.
```

Comment:
Hopefully this is what you both were looking for ^^

Reply:
YES